src/design-system/typography/typography.ts

```diff
@@ -116,8 +132,22 @@ const createTextSize = ({
   };
 };
 
-export const headingSizes = mapValues(typeHierarchy.heading, createTextSize);
-export const textSizes = mapValues(typeHierarchy.text, createTextSize);
+export const headingSizes = Object.entries(typeHierarchy.heading).reduce(
+  (acc, [key, value]) => {
+    acc[key as HeadingKey] = createTextSize(value);
+    return acc;
+  },
+  {} as HeadingSizes
+);
+
+export const textSizes = Object.entries(typeHierarchy.text).reduce(
+  (acc, [key, value]) => {
+    acc[key as TextKey] = createTextSize(value);
+
+    return acc;
+  },
+  {} as TextSizes
+);
```

Comment:
These are computed once on startup and then never again. I think the clarity of lodash outweighs the very few ms this would actually save.

Reply:
Yeah, I close this PR as performance increasing is minimal